patches/chrome-browser-hub-internal-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
-index 91effd3fef959371f9241d9debe31c5618030340..6c71abc61bcd2ae52ee47488d7095d37feca1b99 100644
+index 91effd3fef959371f9241d9debe31c5618030340..f0777c054132dd1ef4fb2d5012c397646b139886 100644
 --- a/chrome/browser/hub/internal/BUILD.gn
 +++ b/chrome/browser/hub/internal/BUILD.gn
-@@ -51,6 +51,7 @@ android_library("java") {
-     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactory.java",
-     "android/java/src/org/chromium/chrome/browser/hub/TranslateHubLayoutAnimationFactoryImpl.java",
+@@ -85,6 +85,7 @@ android_library("java") {
+     "//ui/android:ui_no_recycler_view_java",
+     "//url:url_java",
    ]
-+  sources += brave_browser_hub_internal_java_sources
-   deps = [
-     ":java_resources",
-     "//base:base_java",
++  deps += brave_browser_hub_internal_java_deps
```

Comment:
this is not needed as mentioned above

Reply:
yeah, I addressed at 
https://github.com/brave/brave-core/pull/28679/commits/2796d9bb8afe0220f2637d7b04b7872644627b9c
but I had to add the patch for visibility at
https://github.com/brave/brave-core/pull/28679/commits/2796d9bb8afe0220f2637d7b04b7872644627b9c#diff-6229f2d1f313458d2032010e20a85231507f66cb9e98ad184bd9af95b3b92cc3R9